examples/references/segmentation/pascal_voc2012/code/scripts/training.py

```diff
@@ -1,17 +1,29 @@
 # This a training script launched with py_config_runner
 # It should obligatory contain `run(config, **kwargs)` method
-
+from pathlib import  Path
+import os
 import sys
-from collections.abc import Mapping
-from pathlib import Path
 
+#importing the package parent
+#so no path conflict not happen
+
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
+
+from dataflow.datasets import VOCSegmentationOpencv 
 import torch
 from apex import amp
-from dataflow.datasets import VOCSegmentationOpencv
+#from dataflow.datasets import VOCSegmentationOpencv  # <---- imported before sys.path.insert
```

Comment:
Uncomment and remove comment. Place this line below the line 37

Reply:
done